senders/twilio/twilio.go

```diff
@@ -18,32 +19,48 @@ type sendEventsTwilio interface {
 }
 
 type twilioSender struct {
-	client       *twilio.TwilioClient
+	client       *twilio_client.TwilioClient
 	APIFromPhone string
 	logger       moira.Logger
 	location     *time.Location
 }
 
+// Structure that represents the Twilio configuration in the YAML file
+type Twilio struct {
+	Type          string `mapstructure:"type"`
+	APIAsid       string `mapstructure:"api_asid"`
+	APIAuthToken  string `mapstructure:"api_authtoken"`
+	APIFromPhone  string `mapstructure:"api_fromphone"`
+	VoiceURL      string `mapstructure:"voiceurl"`
+	TwimletsEcho  string `mapstructure:"twimlets_echo"`
+	AppendMessage string `mapstructure:"append_message"`
+}
+
 // Init read yaml config
-func (sender *Sender) Init(senderSettings map[string]string, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
-	apiType := senderSettings["type"]
+func (sender *Sender) Init(senderSettings map[string]interface{}, logger moira.Logger, location *time.Location, dateTimeFormat string) error {
+	var twilio Twilio
+	err := mapstructure.Decode(senderSettings, &twilio)
+	if err != nil {
+		return fmt.Errorf("decoding error from yaml file to twilio structure: %s", err)
+	}
+	apiType := twilio.Type
 
-	apiASID := senderSettings["api_asid"]
+	apiASID := twilio.APIAsid
 	if apiASID == "" {
```

Comment:
Do we need these check if we already have `mapstructure.Decode`?

Reply:
I think that all checks are needed, because by default mapstructure leaves zero value if the field is not filled